### ISSUE TITLE
Upgrade from deprecated Chosen for select lists

### ISSUE THREAD

**Issue by cory-skeers** [uiowa/uiowa#2412]
# Problem

We're currently using the Chosen jQuery library (https://github.com/harvesthq/chosen) to make select lists more user friendly, which has since been deprecated. It is currently functional, but support and updates are no longer being offered.

# Where it is used

**Authenticated:**
- SiteNow node edit forms
- Layout builder styles

**Anonymous:**
- Wellbeing filters
- Academic calendar

# Proposed solution

Look into https://www.drupal.org/project/select2 or https://www.drupal.org/project/select2_all for prepackaged Drupal implementations, https://select2.org/ for jQuery libraries to implement directly through our sitenow profile, or research other Chosen/Select2 options.

**Comment by bspeare** [comment]
Choices is probably the best option here: https://www.drupal.org/project/choices.   There are some style issues that we would need to reset that we encountered in https://github.com/uiowa/uiowa/issues/4897, but it tests well with voiceover.

**Comment by joewhitsitt** [comment]
I assume choices is no longer a security threat? https://github.com/uiowa/uiowa/pull/7883

**Comment by bspeare** [comment]
@joewhitsitt yep, looks fixed: https://www.drupal.org/project/webform/issues/3457416#comment-15715689, https://github.com/Choices-js/Choices/pull/1161.
